docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "2"
+services:
+  slackemon_webserver:
+    build: .
+    restart: always
+    volumes: 
+      - ./:/var/www/html
+    ports:
+      - "80:80"
+
+  slackemon_db:
+    image: postgres:alpine
+    restart: always
+    expose:
+      - "5432"
```

Comment:
This port is accessible only by containers, so no stranger can connect to the DB. If it does not work, it should be changed to ports: - "5432:5432"